docs/sidebars.js

```diff
@@ -67,10 +67,11 @@ module.exports = {
         },
       ],
     },
-    {
-      type: "category",
-      label: "Reference Docs",
-      items: [{ type: "autogenerated", dirName: "api" }],
-    },
+    // there is no 'api' directory?
```

Comment:
This throws an error... There is no 'api' directory?

Reply:
api directory is generated by running `yarn run build` in `docs` package